services/src/pro/apidoc.rs

```diff
@@ -128,7 +132,16 @@ use super::users::{UserCredentials, UserId, UserInfo, UserRegistration, UserSess
         pro::handlers::permissions::remove_permission_handler
     ),
     components(
+        responses(
+            UnsupportedMediaTypeForJsonResponse,
+            PayloadTooLargeResponse,
+            IdResponse,
+            UnauthorizedAdminResponse,
+            UnauthorizedUserResponse,
+            BadRequestQueryResponse
+        ),
         schemas(
+            ErrorResponse,
```

Comment:
why is this unser schemas and not under response?

Reply:
Utoipa trennt zwischen den Traits `ToSchema` und `ToResponse`. `ToSchema` beschreibt nur das Schema des Request Bodies. `ToResponse` beinhaltet jedoch Information über die gesamte Antwort: Unter anderem einen Zeiger auf das Request Body Schema und zum Beispiel den `Content-Type`-Header (standardmäßig "application/json").
Auch wenn `ErrorResponse` im Namen "Response" trägt, implementiert es den `ToSchema`-Trait, da er nur den Body einer Fehlermeldung beschreibt. Die eigentlichen Response-Typen, wie zum Beispiel `UnauthorizedUserResponse`, können dann dieses Schema als Datenformat benutzen.
Da also `ErrorResponse` `ToSchema` implementieren muss, muss er unter "Schemas" stehen.